File: benchmarks/yass/nve.jl
using Pkg
Pkg.activate("./")

using YetAnotherSimulationSuite

i = ARGS[1]
j = parse(Float64, ARGS[2])

bdys = readSystem("../xyzfiles/$(i)au.xyz")

d = Dict(
  "epsilon" => 0.2297,
  "sigma" => 2.95,
  "rs" => 19.0,
  "rc" => 20.0
)

calc = LJ(d)

nve = run(calc, bdys, j * 1u"ps", 1u"fs", NVE())

File: benchmarks/ase/nve.py
from ase.calculators.lj import LennardJones
from ase.io import read
import sys
from ase.md.verlet import VelocityVerlet
from ase import units

i = sys.argv[1]
j = int(sys.argv[2]) * 1000

bdys = read(f"../xyzfiles/{i}au.xyz")

calc = LennardJones(sigma=2.95, epsilon=0.2297, ro=19.0, rc=20.0)

bdys.calc = calc

dyn = VelocityVerlet(bdys, 1 * units.fs)
dyn.run(j)